GAE/src/org/waterforpeople/mapping/app/web/rest/ActionRestService.java

```diff
@@ -207,8 +207,7 @@ private String removeZeroMinMaxValues() {
 
 
     private String publishSurvey(Long surveyId) {
-        SurveyServiceImpl surveyService = new SurveyServiceImpl();
-        surveyService.publishSurveyAsync(surveyId);
```

Comment:
Since you've removed this method, please check whether its used anywhere else and if not lets remove it from the class completely. The entire class should have been removed sooner if not for publishing.

Reply:
It is, but only from PublishSurveyServlet, which I suspect is also unused these days...